<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QMainWindow" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>540</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>540</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="w_MainWidget" native="true">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>541</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QListWidget" name="w_playlist">
     <property name="geometry">
      <rect>
       <x>155</x>
       <y>41</y>
       <width>391</width>
       <height>551</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_close">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>1</y>
       <width>38</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>38</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_min">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>0</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Add">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lab_Cover">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>150</y>
       <width>371</width>
       <height>361</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="w_Interact" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>600</y>
      <width>541</width>
      <height>101</height>
     </rect>
    </property>
    <widget class="QSlider" name="ProgressBar">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>70</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QSlider::groove:horizontal {
    border: none;
    height: 6px;
    border-radius: 3px;
    background: lightgray;
}

QSlider::groove:vertical {
    border: none;
    height: 6px;
    border-radius: 3px;
   /* background: lightgray;*/
	background:tranparent;
}

QSlider::handle:horizontal {
    border: none;
    margin: -5px 0px; /* 上下边距和左右边距*/
    width: 16px;
    height: 16px;
    border-radius: 8px;
    background-color:rgba(240,240,240,0.5);
    border-image: url(:/Icon/SliderPoint.svg);
}
/*划过部分*/
QSlider::sub-page:horizontal {
    background: #8df2f2;
    height: 4px;
    border-radius: 3px;
}
/*未划过部分*/
QSlider::add-page:horizontal {
    background: lightgray;
    height: 4px;
    border-radius: 3px;
}
/*
QSlider 
{
	background-color: rgba(22, 22, 22, 0.7);
	padding-top: 15px;  
	padding-bottom: 15px;
	border-radius: 5px; 
}




QSlider::add-page:vertical 
{
	background-color: rgba(240,240,240,0.6);
	width:5px;
	border-radius: 2px;
}
 
QSlider::sub-page:vertical 
{
	background-color: rgba(240,240,240,0.6);
	width:5px;
	border-radius: 2px;
}
 

QSlider::handle:vertical
{
	height: 14px;  
	width: 14px;
	margin: 0px -4px 0px -4px;
	border-radius: 7px;
	background: white;
}

QSlider::groove:honrizontal
{
	background-color:rgba(240,240,240,0.6);
	width:8px;
}
*/
 
</string>
     </property>
     <property name="tracking">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lab_TotalTime">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>70</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>4:00</string>
     </property>
    </widget>
    <widget class="QLabel" name="lab_CurrentTime">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>27</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>26</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>0:00</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_cover">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_play_or_pause">
     <property name="geometry">
      <rect>
       <x>274</x>
       <y>6</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton#pButtonOk:hover
{
    color:white;
    background-color:rgb(44 , 137 , 255);
    border: 1px solid rgb(11 , 137 , 234);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_next">
     <property name="geometry">
      <rect>
       <x>335</x>
       <y>6</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_volumn">
     <property name="geometry">
      <rect>
       <x>397</x>
       <y>6</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_prev">
     <property name="geometry">
      <rect>
       <x>213</x>
       <y>6</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_playlist">
     <property name="geometry">
      <rect>
       <x>463</x>
       <y>6</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_loop">
     <property name="geometry">
      <rect>
       <x>147</x>
       <y>6</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lab_MusicInfo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>120</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>10</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>100</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>11</pointsize>
       <bold>true</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
color: rgba(10,10,10,0.9);}
</string>
     </property>
     <property name="text">
      <string>暂无歌曲哦</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextInteractionFlag::NoTextInteraction</set>
     </property>
    </widget>
   </widget>
   <zorder>w_Interact</zorder>
   <zorder>w_MainWidget</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
